frontend/src/components/pages/teacher/DisplayClassroomPage/index.tsx

```diff
@@ -71,9 +71,9 @@ const studentFormDefaultValues: StudentForm = {
 };
 
 const classroomFormDefaultValues: ClassroomForm = {
-  className: "",
-  startDate: undefined,
-  gradeLevel: undefined,
+  className: null,
+  startDate: null,
+  gradeLevel: null,
```

Comment:
does it matter that we send `null` to BE if gradeLevel is not set?

Reply:
Since the data is validated on the frontend I don't think we should ever actually be sending null. Did you see a case where this could happen?

Reply:
oops i was lacking sleep, idk why i thought gradeLevel was optional or smth

Reply:
Nw im also dead